dev/packages/alpha/nginx/0.0.3/dataset/ingress_controller/manifest.yml

```diff
@@ -0,0 +1,24 @@
+title: Nginx ingress_controller logs
+type: logs
+release: beta
+streams:
+- input: logs
+  enabled: false
```

Comment:
Somehow it seems the UI still does not pick up this one :-( Could you raise this with the KB team?

Reply:
It's already raised: https://github.com/elastic/kibana/issues/64063